pulsar-broker/src/test/java/org/apache/pulsar/broker/namespace/NamespaceUnloadingTest.java

```diff
@@ -68,4 +75,26 @@ public void testUnloadPartiallyLoadedNamespace() throws Exception {
         producer.close();
     }
 
+    @Test
+    public void testUnloadWithTopicCreation() throws PulsarAdminException, PulsarClientException {
```

Comment:
The test can be passed without applying the fix.

Reply:
You can run this test some times. It can't 100% reproduce it at one time.

Reply:
+1. Still can't reproduce after retrying multiple times.

Reply:
You can add loop counts and partition numbers to help increase the possibility.  :)

Reply:
I added partition numbers for this test.